File: Back/src/ProEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;

namespace ProEventos.Domain
{
    public class Evento
    {
        public int id { get; set; }
        public string imagemURL { get; set; }
        public string local { get; set; }
        public DateTime? dataEvento { get; set; }
        public string tema { get; set; }
        public int qtdPessoas { get; set; }
        public string telefone { get; set; }
        public string email { get; set; }
        public IEnumerable<Lote> lotes { get; set; }
        public IEnumerable<RedeSocial> redesSociais { get; set; }
        public IEnumerable<PalestranteEvento> palestrantesEventos { get; set; }

    }
}

File: Front/ProAgil-APP/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento {
   id: number;
   imagemURL: string;
   local: string;
   dataEvento: Date;
   tema: string;
   qtdPessoas: number;
   telefone: string;
   email: string;
   lotes: Lote[];
   redesSociais: RedeSocial[];
   palestrantesEventos: Palestrante[];
}
